realsense2_camera/src/realsense_node_factory.cpp

```diff
@@ -211,11 +218,13 @@ void RealSenseNodeFactory::onInit()
 		ros::NodeHandle nh = getNodeHandle();
 		auto privateNh = getPrivateNodeHandle();
 		privateNh.param("serial_no", _serial_no, std::string(""));
-    	privateNh.param("usb_port_id", _usb_port_id, std::string(""));
-    	privateNh.param("device_type", _device_type, std::string(""));
+		privateNh.param("usb_port_id", _usb_port_id, std::string(""));
+		privateNh.param("device_type", _device_type, std::string(""));
 
 		std::string rosbag_filename("");
 		privateNh.param("rosbag_filename", rosbag_filename, std::string(""));
```

Comment:
Looks like we are dumping rosbag playback support, but also some code was ported here and abandoned. Can you clean up? If we don't call oldOnInit, can just remove. Ideally, porting the rosbag playback to onInit

Reply:
which code do you think was ported by abandoned?

Reply:
are you talking about reset request publisher and replacement of exit to requestRosShutdown?